test/integration/redirects_test.rb

```diff
@@ -77,4 +78,67 @@ def test_destroy_article_put
     assert_equal(article_url(Article.first.id),
                  @response.header["Location"])
   end
+
+  #######
+
+  TERM_ID = GlossaryTerm.take.id
+  GLOSSARY_USER_LOGIN = User.take.login
```

Comment:
**Should I use `.take` or instead pick a particular object?**
I used .take to try to express the intent that we shouldn't care which GlossaryTerm or User is used in the tests. But -- of course - that might make the tests more random.

Reply:
Again, FWIW, in my opinion, randomness in tests invites trouble in the form of some spurious unexpected failure down the line long after we've forgotten all about this.  Those are so hard to debug.  And time may be limited at the time the failure occurs for any of a thousand unforeseeable reasons.